build.gradle

```diff
@@ -1327,12 +1327,14 @@ project(':connect:runtime') {
   archivesBaseName = "connect-runtime"
 
   dependencies {
+
     compile project(':connect:api')
-    compile project(":connect:transforms")
     compile project(':clients')
     compile project(':tools')
-    compile libs.slf4jApi
+    compile project(':connect:json')
+    compileOnly project(':connect:transforms')
```

Comment:
Changing this to `compileOnly` breaks https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/tools/TransformationDoc.java#L88 which is used for autogenerating the transformation docs.